tests/telemetry/test_telemetry.py

```diff
@@ -40,11 +60,9 @@ def test_opt_str_validation():
     assert type(exception_raised) == TypeError
 
 
-# Test class functions
-def test_check_stats_enabled():
+def test_check_stats_enabled(ignore_env_var_and_set_tmp_default_home_dir):
     stats_enabled = telemetry.check_stats_enabled()
-    assert isinstance(stats_enabled, bool)
-    assert stats_enabled
+    assert stats_enabled is True
```

Comment:
What's wrong with asserting the inner value?

Reply:
nothing wrong but `statis_enabled is True` covers the previous two lines (is bool and is True), it's just shorter and more explicit :)